base_export_manager/models/res_users.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 - Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+
+from openerp import api, models
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    @api.v7
+    def get_export_models(self, cr, uid):
+        cr.execute("SELECT model "
+                   "FROM ir_model "
```

Comment:
This operation is not so complicated and perf-killer for being done in ORM, so I prefer to use ORM